uk.co.compucorp.civicrm.hrleaveandabsences/views/my-leave/components/my-leave-request.html

```diff
@@ -172,6 +179,43 @@ <h4 class="modal-title">Leave Request</h4>
       </uib-tabset>
     </div>
   </div>
+
+  <div ng-show="modal.leaveRequest.isAdmin"
+       class="row chr_leave-request-modal__response">
+    <div class="col-xs-12 form-horizontal">
+      <div class="form-group">
+        <label class="col-xs-4 chr_leave-request-modal__response-label">Response<span>*</span></label>
+        <div class="col-xs-8 chr_leave-request-modal__response-dropdown">
+          <div class="chr_custom-select chr_custom-select--full">
+            <select ng-model="modal.leaveRequest.selectedResponse">
+              <option value="1">- select -</option>
+              <option value="2">Approve</option>
+              <option value="3">Reject</option>
+              <option value="4">More Information Required</option>
+              <option value="5">Cancel</option>
+            </select>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div ng-show="modal.leaveRequest.selectedResponse == 3">
+      <div class="col-xs-12">Please add a comment:</div>
+      <div class="col-xs-12 chr_leave-request-modal__comment"
```

Comment:
I think both `.chr_leave-request-modal__comment` and `.chr_leave-request-modal__add-comment` should be more generic and not tightly related to the leave request modal, as this wysiwyg+action is gonna be a reusable component. Something like 
```scss
.chr_wysiwyg {

}

// Note that it is "action" and not "comment"
// as the label for the action could be anything
.chr_wysiwyg__action {

}